docs/ExpectAPI.md

```diff
@@ -333,10 +333,10 @@ describe('not.stringContaining', () => {
 });
 ```
 
-### `expect.not.stringMatching(regexp)`
+### `expect.not.stringMatching(string | regexp)`
 
-`expect.not.stringMatching(regexp)` matches any received string that does not
-match the expected regexp.
+`expect.not.stringMatching(string | regexp)` matches the received string that does not
+match the expected string or regexp.
```

Comment:
I think we should keep the `string | regexp` update but remove the "string or" comment,

Reply:
That's because while the arg types can be either a string or a regex, the matching is done via regexThat means you can do this:

```js
it("the string matches as a regex", () => {
  expect(["foo bar"]).toEqual([expect.stringMatching("bar$")]);
});

it("the string matches substrings because regex", () => {
  expect(["foo ba"]).toEqual([expect.stringMatching("bar")]);
});
```

Reply:
I agree. I will change this one and the other one below as well